exasol_advanced_analytics_framework/event_handler/event_handler_result.py

```diff
@@ -0,0 +1,9 @@
+from dataclasses import dataclass
+from typing import Optional, List
+
+
+@dataclass(unsafe_hash=True)
```

Comment:
what does this do?

Reply:
this is result object holds what the event handler should return

Reply:
I mean the unsafe hash

Reply:
aha it is not a crucial stuff. I did it like that because I thought the result object should be comparable and mutable. But comparability might be not necessary.